CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+## Update from upstream
```

Comment:
Good to move this. What about adding a contributing section to the readme and linking here?

Reply:
Good idea 👍